src/utils/givethIoSyncer.js

```diff
@@ -0,0 +1,60 @@
+const config = require('config');
+const Queue = require('bull');
+const logger = require('winston');
+
+const { errorMessages } = require('./errorMessages');
+const { CampaignStatus } = require('../models/campaigns.model');
+
+const updateCampaignQueue = new Queue('trace-campaign-updated', {
+  redis: config.get('sharedRedis'),
+});
+const updateGivethIoProjectQueue = new Queue('givethio-project-updated', {
+  redis: config.get('sharedRedis'),
+});
+
+const addVerifiedCampaignUpdateEvent = ({
+  campaignId,
+  status,
+  givethIoProjectId,
+  title,
+  description,
+}) => {
+  // giveth.io will handle this event
+  updateCampaignQueue.add({ campaignId, status, givethIoProjectId, title, description });
+};
+
+const initHandlingGivethIoUpdateEvents = app => {
+  updateGivethIoProjectQueue.process(1, async (job, done) => {
+    try {
+      // There are title, description in job.data but we dont use theme right now
+      const { campaignId, verified, archived } = job.data;
```

Comment:
is `verified` part of the event from givethio?

Reply:
Sure when project `verified` will change in adminBro, the verified would change in giveth trace,
this change is one way, only giveth.io can change the `verified`